.changeset/fruity-brooms-stare.md

```diff
@@ -0,0 +1,5 @@
+---
+"@nomicfoundation/hardhat-foundry": patch
+---
+
+When running forge install, check the --no-commit flag is supported on installed forge version
```

Comment:
Changesets don't need to include implementation details. They should only tell the user what changed.

```suggestion
Fixed a bug in the `init-foundry` task that prevented it from being used with the latest versions of `forge`
```